src/main/kotlin/org/partiql/scribe/sql/SqlDialect.kt

```diff
@@ -173,7 +173,7 @@ abstract class SqlDialect : AstBaseVisitor<SqlBlock, SqlBlock>() {
 
     override fun visitTypeReal(node: Type.Real, tail: SqlBlock): SqlBlock = tail concat "REAL"
 
-    override fun visitTypeFloat32(node: Type.Float32, tail: SqlBlock): SqlBlock = tail concat "FLOAT32"
+    override fun visitTypeFloat32(node: Type.Float32, tail: SqlBlock): SqlBlock = tail concat "REAL"
```

Comment:
Can we port these changes to `visitTypeFloat32` and `visitTypeCustom` over to `SqlDialect`?

Reply:
Ok. https://github.com/partiql/partiql-lang-kotlin/issues/1494